test/terra/noise/test_noise_model.py

```diff
@@ -335,6 +338,22 @@ def test_can_run_circuits_with_delay_noise(self):
         result = AerSimulator().run(qc, noise_model=noise_model).result()
         self.assertTrue(result.success)
 
+    def test_from_dict(self):
+        noise_ops_1q = [((IGate(), [0]), 0.9),
+                     ((XGate(), [0]), 0.1)]
+
+        noise_ops_2q = [((PauliGate('II'), [0, 1]), 0.9),
+                     ((PauliGate('IX'), [0, 1]), 0.045),
+                     ((PauliGate('XI'), [0, 1]), 0.045),
+                     ((PauliGate('XX'), [0, 1]), 0.01)]
+
+        noise_model = NoiseModel()
+        with self.assertWarns(DeprecationWarning):
+            noise_model.add_quantum_error(QuantumError(noise_ops_1q, 1), 'h', [0])
+            noise_model.add_quantum_error(QuantumError(noise_ops_1q, 1), 'h', [1])
+            noise_model.add_quantum_error(QuantumError(noise_ops_2q, 2), 'cx', [0, 1])
+            noise_model.add_quantum_error(QuantumError(noise_ops_2q, 2), 'cx', [1, 0])
+            noise_model.from_dict(noise_model.to_dict())
```

Comment:
I think there might be an assertion missing from this test? At the moment it's testing that no exception is thrown, but not that the model is actually correct.  Is it meant to be the case that `noise_model == NoiseModel.from_dict(noise_model.to_dict())` or something?

(Also, `noise_model.from_dict` looks odd instead of `NoiseModel.from_dict` for the intent of the function, but that's not a big deal.)

Reply:
Thanks. I updated this test case in 4981575.